src/test/resources/records/GenericRecord.java

```diff
@@ -0,0 +1,3 @@
+package records;
+
+public record GenericRecord<T>(T a, T b) {}
```

Comment:
I think this line is syntactically incorrect. Instead of `record`, there should be `class`, `interface`, or even `enum`. I tried running the tests after replacing `record` with `class` and it returned the expected string.